covid_graphs/covid_graphs/fit_atg_model.py

```diff
@@ -26,7 +26,7 @@ def fit_atg_model(xs: np.ndarray, ys: np.ndarray) -> AtgModelFit:
     a_init = 2000.0
     tg_init = 7.0
     exp_init = 6.23
-    t0_init = 0.0
+    t0_init = xs[0]
```

Comment:
This makes the optimisation independent of the chosen conversion of dates to floats.